src/pages/journal/journal-components/journal-candidate/journal-candidate.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input } from '@angular/core';
+import { Name } from '../../../../common/domain/DJournal';
+
+@Component({
+  selector: 'journal-candidate',
+  templateUrl: 'journal-candidate.html'
+})
+export class JournalCandidateComponent {
```

Comment:
Looking at the name of this component I'm not sure what is it doing. We could easily call this component as `CandidateInfoHighlightsComponent` for example and then we'll have a better understanding of the problem that it solves.